aip_xx1_launch/launch/pointcloud_preprocessor.launch.py

```diff
@@ -37,17 +37,32 @@ def launch_setup(context, *args, **kwargs):
     )
 
     # set concat filter as a component
-    concat_component = ComposableNode(
-        package="autoware_pointcloud_preprocessor",
-        plugin="autoware::pointcloud_preprocessor::PointCloudConcatenateDataSynchronizerComponent",
-        name="concatenate_data",
-        remappings=[
-            ("~/input/twist", "/sensing/vehicle_velocity_converter/twist_with_covariance"),
-            ("output", "concatenated/pointcloud"),
-        ],
-        parameters=[concatenate_and_time_sync_node_param],
-        extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
-    )
+    if LaunchConfiguration("use_cuda_preprocessor").perform(context):
+        concat_component = ComposableNode(
+            package="autoware_cuda_pointcloud_preprocessor",
+            plugin="autoware::cuda_pointcloud_preprocessor::CudaPointCloudConcatenateDataSynchronizerComponent",
+            name="concatenate_data",
+            remappings=[
+                ("~/input/twist", "/sensing/vehicle_velocity_converter/twist_with_covariance"),
+                ("output", "concatenated/pointcloud"),
+                ("output/cuda", "concatenated/pointcloud/cuda"),
+            ],
+            parameters=[concatenate_and_time_sync_node_param],
+            # NOTE(knzo25): when using  the cuda blackboard, this setting can not be made global
+            # extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
+        )
+    else:
+        concat_component = ComposableNode(
+            package="autoware_pointcloud_preprocessor",
+            plugin="autoware::pointcloud_preprocessor::PointCloudConcatenateDataSynchronizerComponent",
+            name="concatenate_data",
+            remappings=[
+                ("~/input/twist", "/sensing/vehicle_velocity_converter/twist_with_covariance"),
+                ("output", "concatenated/pointcloud"),
+            ],
+            parameters=[concatenate_and_time_sync_node_param],
+            extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
+        )
```

Comment:
The two branches differ only in their package, plugin and extra_params. The rest of the code could be merged into one to prevent future contributors from only modifying the one they're concerned about

Reply:
Addressed in [bfcf9a2](https://github.com/tier4/aip_launcher/pull/348/commits/bfcf9a2a563dadc3ef4d6c96e4dc60049314c392)